webpages/reports/pubsbio.php

```diff
@@ -4,12 +4,13 @@
 $report['name'] = 'Pubs - Participant Bio and pubname';
 $report['multi'] = 'true';
 $report['output_filename'] = 'pubs_bios.csv';
-$report['description'] = 'Show the id, pubsname and bio for each participant who is on at least one scheduled, public session.';
-$report['categories'] = array(
+$report['description'] =
+    'Show the id, pubsname and bio for each participant who is on at least one scheduled, public session.';
```

Comment:
Please put this back on the previous line. That's how all the other reports are done and I would like a consistent look.

Reply:
Okay.